src/lib/credentials.py

```diff
@@ -43,8 +45,13 @@ def get_dynatrace_api_key_from_env():
     return os.environ.get(_DYNATRACE_ACCESS_KEY_SECRET_NAME, None)
 
 
-def get_dynatrace_url_from_env():
-    return os.environ.get(_DYNATRACE_URL_SECRET_NAME, None)
+def get_dynatrace_log_ingest_url_from_env():
+    # TODO alternatively fetch from GCP secret
```

Comment:
todo ?

Reply:
I didn't wanted to introduce too much changes in this PR. I'll create separate task to implement it as I suspect it will require more code changes

Reply:
If I understand correctly fetch_secret() checks if env variable is set (if yes returns the value if not secret is fetched from secret manager) -> get_dynatrace_log_ingest_url_from_env() could just call fetch_secret() ?

Reply:
I confirmed that fetch_secret() is (officially) used only by function. I'll remove TODO, it is not needed